### PR TITLE
Update Hacktoberfest for 2022

### PR DESCRIPTION
**Pull request by MattIPv4**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [ ] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> Updating the Hacktoberfest logo to be the 2022 branding that was published today. Also updating the Hacktoberfest website to the new domain being used for the 2022 season, and using the updated description wording used for Hacktoberfest.

_I am affiliated with Hacktoberfest -- [my PRs](https://github.com/github/explore/pulls/MattIPv4) have historically been accepted here for updating Hacktoberfest each year_

### CODE REVIEW

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Comment by MattIPv4** [comment]
Hm, not sure I'm gonna get the CoI check green unless I lie in the PR description.

Looking at the other failures now 👀

File: topics/hacktoberfest/index.md [hunk 1]
```diff
@@ -3,8 +3,8 @@ created_by: DigitalOcean and GitHub
...
 logo: hacktoberfest.png
 released: October 2014
-short_description: Hacktoberfest is a month-long celebration of open source software.
+short_description: Hacktoberfest is a a month-long celebration of open-source projects, their maintainers, and the entire community of contributors.
 topic: hacktoberfest
-url: https://hacktoberfest.digitalocean.com/
+url: https://hacktoberfest.com/
 ---
-Hacktoberfest is a month-long celebration of open source software. Each October, open source maintainers give new contributors extra attention as they guide developers through their first pull requests on GitHub.
+Hacktoberfest is a a month-long celebration of open-source projects, their maintainers, and the entire community of contributors. Each October, open source maintainers give new contributors extra attention as they guide developers through their first pull requests on GitHub.
```
Comment (kenyonj) [inline topics/hacktoberfest/index.md]: Adding the hyphen here is causing a failure in the tests. Can we revert it?
```suggestion
Hacktoberfest is a a month-long celebration of open source projects, their maintainers, and the entire community of contributors. Each October, open source maintainers give new contributors extra attention as they guide developers through their first pull requests on GitHub.
```

**Review by kenyonj — CHANGES_REQUESTED** [review]
There are two CI failures. One I have a suggested change to fix and the other is the image needs to be smaller. Once these are fixed we can merge this in.

**Review by kenyonj — APPROVED** [review]

**Comment by MattIPv4** [comment]
Thank you 💙

**Pull request merged**